__tests__/app.test.js

```diff
@@ -76,106 +76,159 @@ describe("Reviews TESTS", () => {
+              review_id: 1,
+              title: "Agricola",
+              designer: "Uwe Rosenberg",
+              owner: "mallionaire",
+              review_img_url:
+                "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+              review_body: "Farmyard fun!",
+              category: "euro game",
+              created_at: expect.any(String),
+              votes: 6,
+            });
+          });
+      });
     });
   });
-});
-
-describe('Users TESTS', () => {
-  describe('GET/users', () => {
-    test('Status 200, responds with  an array of objects, each object should have the following properties: username, name, avatar_url ', () => {
+  describe("GET /api/reviews", () => {
+    test("200 response, returns an object containing array of objects which contains following properties ", () => {
       return request(app)
-        .get("/api/users")
+        .get("/api/reviews")
         .expect(200)
-        .then((result) => {
-          const userArr = result.body.users;
-          expect(userArr).toBeInstanceOf(Array);
-          expect(userArr.length).toBe(4);
-          userArr.forEach((category) => {
-            expect(category.hasOwnProperty("username")).toBe(true);
-            expect(category.hasOwnProperty("name")).toBe(true);
-            expect(category.hasOwnProperty("avatar_url")).toBe(true);
+        .then(({ body: { reviews } }) => {
+          expect(reviews.length).toBeGreaterThan(0);
+          reviews.forEach((review) => {
+            expect(review).toEqual(
+              expect.objectContaining({
+                review_id: expect.any(Number),
+                title: expect.any(String),
+                category: expect.any(String),
+                designer: expect.any(String),
+                owner: expect.any(String),
+                review_body: expect.any(String),
+                review_img_url: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                comment_count: expect.any(Number),
+              })
+            );
           });
         });
     });
-  });
-});
-describe("ERROR's", () => {
-  describe('GET error handling', () => {
-    test("status: 400, Bad Request", () => {
-      return request(app)
-        .get("/api/reviews/not an ID")
-        .expect(400)
-        .then(({ body: { msg } }) => {
-          expect(msg).toBe("wrong input");
-        });
-    });
-    test("status: 404, Not Found", () => {
+    test("objects are sorted into descending order by date", () => {
       return request(app)
-        .get("/api/reviews/10000000")
-        .expect(404)
-        .then(({ body: { msg } }) => {
-          expect(msg).toBe("ID NOT FOUND");
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toBeSortedBy("created_at", {
+            descending: true,
+            coerce: true,
+          });
```

Comment:
reviews should also be able to accept a query and be sorted by category - i can't see the tests to check for this, can this be implemented please :)